src/lib.rs

```diff
@@ -17,9 +17,6 @@
 
 // Re-exports
 pub use algonaut_client as client;
-pub use algonaut_client::algod::Algod;
```

Comment:
Is it also a good time to implement the renaming you suggested in another issue (`AlgodBuilder`) and the restructuring of the modules `AlgodBuilder::v1::client()` instead of `AlgodBuilder::client_v1()`?

Reply:
The abstraction layer for the clients (https://github.com/manuelmauro/algonaut/issues/16#issuecomment-833354817), yes why not.